src/platform/silabs/platformAbstraction/WiseMcuSpam.cpp

```diff
@@ -53,12 +60,13 @@ void soc_pll_config(void);
 #endif
 // TODO Remove this when SI91X-16606 is addressed
 #ifdef SI917_DEVKIT
-#define SL_LED_COUNT 1
-uint8_t ledPinArray[SL_LED_COUNT] = { SL_LED_LEDB_PIN };
+#define SL_LED_COUNT SL_SI91X_RGB_LED_COUNT
```

Comment:
what si91x-16606 addresed?

Reply:
LED0 instance has been added on Devkit instead of LEDB/G/R instance.

Reply:
yes , will remove and push the commit. Thanks for the comment